src/shashmap.rs

```diff
@@ -29,10 +28,10 @@ impl Shashmap {
     /// * `tx` - `Transaction` where inputs are inserted, and outputs are removed
     pub fn unspend_transaction(&mut self, _tx: &Transaction) {}
 
-    /// Return the `Block` id based on `Slip`
+    /// Return the `Block` id based on `OutputSlip`
```

Comment:
Comment doesn't make sense under proposed changes. Should be "Return `OutputSlip` from `SlipID`"